requirements_test.txt

```diff
@@ -38,3 +38,10 @@ spyking-circus
 
 ## herdingspikes
 herdingspikes
+
+## mearec
+mearec
+
+## others
+docker
+spython
```

Comment:
I don't think we will install images on our test suite it will be too slow.

Reply:
ok, I changed this into a comment, as I think it should still be listed for completeness and manual testing.